analyses/snv-callers/scripts/03-merge_callers.R

```diff
@@ -0,0 +1,304 @@
+# Merge the caller VAF and TMB files
+#
+# 2019
+#
+# C. Savonen for ALSF - CCDL
+#
+# Purpose: Merge callers' TMB and VAF files into total files with a column `caller`
+# to designate their origin.
+
+# Files Output:
+# "all_callers_vaf.<file_format>" - contains all the VAF file information for all callers.
+# "all_callers_tmb.<file_format>" - contains all the TMB file information for all callers.
+# "mutation_id_list.<file_format>" - a full list of the mutations that can be
+#                                    used for an UpSetR graph
+# "callers_per_mutation.<file_format>" - contains a breakdown for each mutation of what callers
+#                                        called it. Will be used to identify the consensus mutations.
+
+# Option descriptions
+# --vaf : Parent folder containing the vaf and tmb files for each folder.
+#                                             <caller_name>_vaf.<file_format>
+#                                             <caller_name>_tmb.<file_format>
+# --file_format: What type of file format were the vaf and tmb files saved as? Options are
+#               "rds" or "tsv". Default is "rds".
+# --output : Where you would like the output from this script to be stored.
+# --overwrite : If TRUE, will overwrite any reports of the same name. Default is
+#              FALSE
+#
+#
+# Command line example:
+#
+# Rscript 03-merge_callers.R \
+# -v results \
+# -o strelka2 \
+# -s wxs \
+# -w
```

Comment:
What does this `-w` correspond to?

Reply:
I used to have `--overwrite` have a short flag but then I deleted it. Forgot to delete it from the example.